cmd/trace-agent/remote_config.go

```diff
@@ -86,3 +101,18 @@ func remoteConfigHandler(r *api.HTTPReceiver, client pbgo.AgentSecureClient, tok
 
 	})
 }
+
+func getContainerTags(req *http.Request, cfg *config.AgentConfig) []string {
+	if cfg == nil || cfg.ContainerTags == nil {
+		_ = log.Warn("ContainerTags not configured")
```

Comment:
If there's no containerID with the request and container tags are not expected it looks like this log will be spammy

Reply:
Removed the log line in https://github.com/DataDog/datadog-agent/pull/12034/commits/d42f94ef2f09c91b841032e8965256dec0337a40